doc/method/discussion.md

```diff
@@ -65,6 +65,43 @@ common and unique heuristics to automated, scalable name-string matching.
 
 ## Potential improvements and future work
 
-Certainly the taxonomy could be improved by incorporating DUA-encumbered
-sources such the IUCN Red List, but doing so would conflict with the
-project's open data requirement.
+The development of the assembly asoftware has been driven by the needs
+of the Open Tree project, not by any concerted effort to create a
+widely applicable or theoretically principled tool.  Many improvements
+are possible on both practical and theoretical grounds.  Following are
+some of the directions for development that could have the highest
+impact.
+
+* It is very likely that the method could be improved by making better
+  use of tree topology, and decreasing its reliance on the names of
```

Comment:
No, I meant the topology of the taxonomy. Maybe say "classification" or "hierarchy" instead of "tree topology". Will revisit.